test/PushNotificationTests/BaseTestSuite.cpp

```diff
@@ -113,6 +114,27 @@ void BaseTestSuite::ChannelRequestUsingRemoteId()
     }
 }
 
+void BaseTestSuite::ChannelRequestCheckExpirationTime()
+{
+    if (PushNotificationManager::Default().IsSupported())
+    {
+        auto channelOperation{ PushNotificationManager::Default().CreateChannelAsync(c_azureRemoteId) };
+        VERIFY_SUCCEEDED(ChannelRequestHelper(channelOperation));
+
+        auto channel{ channelOperation.GetResults().Channel() };
+        auto expirationTime{ channel.ExpirationTime() };
+        auto expiryBound{ winrt::clock::now() + std::chrono::days(31) };
+
+        // Need to add 30 days to match expiration time.
+        VERIFY_IS_LESS_THAN(expirationTime, expiryBound);
+    }
+    else
+    {
+        auto channelOperation{ PushNotificationManager::Default().CreateChannelAsync(c_azureRemoteId) };
```

Comment:
How was the negative test verified?

Reply:
In our negative tests, we return E_FAIL. This is not ExpirationTime specific but moreso to cover unsupported WAS scenarios.
https://github.com/microsoft/WindowsAppSDK/blob/77761e244289fda6b3d5f14c7bded189fed4fb89/dev/PushNotifications/PushNotificationManager.cpp#L208

Reply:
I get that but did we run the negative test case too or is this just eye-balling :)

Reply:
Just eye-balling the negative test. Since this is a common negative test that we have trusted in the past (downlevel testing), we should be ok.